MTM-62312: Fix failure of tenant creation with tenant policy that contains override of default tenant option (#2965)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025.72.0-fix-failure-when-tenant-with-tenant-policy.md

---

> date: 30.02.2025

Reviewer: ```suggestion
date:
```

Reviewer: Nice date :-). Actually we anyway leave the date field empty.

> title: Fix failure of tenant creation with tenant policy that contains override of default tenant option 

Reviewer: ```suggestion
title: Tenant creation with tenant policy overriding the default tenant option works properly
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-JlFdtOPva
    label: Rest API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-62312
version: 2025.72.0
---

> When enterprise tenant create a subtenant with tenant policy that tries to override the default tenant option will now

Reviewer: ```suggestion
Previously, when an {{< enterprise-tenant >}} created a subtenant with a tenant policy that tried to override the default tenant option the tenant creation failed. This issue has been fixed an creating a subtenant with a tenant policy overriding the default tenant option now works properly.
```
